app/javascript/components/app.jsx

```diff
@@ -58,6 +58,11 @@ persistCache({
 const client = new ApolloClient({
   link,
   cache,
+  defaultOptions: {
+    query: {
+      fetchPolicy: 'network-only',
```

Comment:
I would think `'cache-and-network'` would be preferable, no? This will never hit the cache...

Reply:
https://www.apollographql.com/docs/react/api/react-apollo/#optionsfetchpolicy

Nope. Just first request. Then we're living the caching dream.

Reply:
agreed, `cache-and-network` seems like the play.